src/sonic-config-engine/tests/test_multinpu_cfggen.py

```diff
@@ -137,25 +137,25 @@ def test_backend_asic_portchannels(self):
     def test_frontend_asic_portchannel_mem(self):
         argument = "-m {} -p {} -n asic0 --var-json \"PORTCHANNEL_MEMBER\"".format(self.sample_graph, self.port_config[0])
         output = json.loads(self.run_script(argument))
-        self.assertListEqual(output.keys(), \
+        self.assertListEqual(list(output.keys()), \
```

Comment:
```suggestion
        self.assertListEqual(list(output), \
```

Reply:
Just curious what list(dict) would render as the suggested change appears to change semantics.

Reply:
Will leave this one as is as the asserListEqual would fail if the arg are of different type

Reply:
I believe `list(mydict.keys())` is the proper way to obtain a list of keys in Python 3. As of Python 3, `.keys()` returns an object of type `dict_keys()`, which can be cast/converted to a list. Reference: https://www.tutorialkart.com/python/python-dictionary-get-keys-as-list/

Reply:
@jleveque this is correct the type is different dict_keys in Python3 while list in Python2 for .keys() call. Removing the list will fail the AssertListEqual

Reply:
@tahmed-dev: You're agreeing with me :)

Reply:
@jleveque Yes. This what I also mentioned a couple of comments above.

> Will leave this one as is as the `assertListEqual` would fail if the arg are of different type